test/api/james.spec.js

```diff
@@ -346,10 +346,11 @@ describe('The James APIs', function() {
 
       api.storeDlpRules(domainId, rules)
         .then(() => {
-          const getRequestHistory = setup.mockAdapter.history.put[0];
+          const putRequestHistory = setup.mockAdapter.history.put[0];
 
-          expect(getRequestHistory.url).toBe(url);
-          expect(getRequestHistory.method).toBe('put');
+          expect(putRequestHistory.url).toBe(url);
+          expect(putRequestHistory.method).toBe('put');
+          expect(JSON.parse(putRequestHistory.data)).toEqual(rules);
```

Comment:
should expect deep equal with array?

Reply:
There's no 'to.deepEqual` in Jest, `toEqual` is already doing that.

> Use .toEqual to compare recursively all properties of object instances (also known as "deep" equality). It calls Object.is to compare primitive values, which is even better for testing than === strict equality operator.

(https://jestjs.io/docs/en/expect#toequalvalue)

Reply:
Ok. Good to know.